src/modules/unit/components/UnitsOnMap.js

```diff
@@ -1,28 +1,31 @@
+/* eslint-disable react/prop-types */
```

Comment:
There is a lot of these, maybe just disable this eslint rule?

Reply:
There are following rationales to disable these locally:

1. The project uses Airbnb's ESLint where the intended behavior originates
2. React upgrade from 15 to 16 is planned but not started and the logic for PropTypes will go through some changes then if not earlier
3. The project has various conventions for syntax, most notably Flow which interferes with PropTypes. Flow will be changed to TypeScript and bring changes
4. The PR is rather large but it aims to change the actual React source code as little as possible (cf. 2 and 3)
5. The plan is to gradually migrate the source to conform Airbnb's "vanilla" ESLint when an individual file is subject to change in the upcomping feature implementation